agent/api/container/container.go

```diff
@@ -58,6 +58,14 @@ const (
 	// MetadataURIFormat defines the URI format for v3 metadata endpoint
 	MetadataURIFormat = "http://169.254.170.2/v3/%s"
 
+	// MetadataURIEnvironmentVariableNameV4 defines the name of the environment
+	// variable in containers' config, which can be used by the containers to access the
+	// v4 metadata endpoint
+	MetadataURIEnvironmentVariableNameV4 = "ECS_CONTAINER_METADATA_URI_V4"
```

Comment:
prefer using short var names in go, this can be `MetadataURIEnvVarNameV4`

Reply:
sure!